src/components/MultiHopTrade/components/SharedTradeInput/SharedTradeInput.tsx

```diff
@@ -44,23 +45,26 @@ export const SharedTradeInput: React.FC<SharedTradeInputProps> = ({
       justifyContent='center'
       maxWidth={isCompact || isSmallerThanXl ? '500px' : undefined}
     >
-      <Center width='inherit'>
-        <Card
-          flex={1}
-          width='full'
-          maxWidth='500px'
-          ref={tradeInputRef}
-          as='form'
-          onSubmit={onSubmit}
-        >
-          <SharedTradeInputHeader
-            initialTab={tradeInputTab}
-            rightContent={headerRightContent}
-            onChangeTab={onChangeTab}
-          />
-          {bodyContent}
-          {footerContent}
-        </Card>
+      <Center width='inherit' alignItems='flex-start'>
+        <Box width='full' maxWidth='500px'>
+          <ThorFreeFeeBanner />
```

Comment:
preferably blocking: it should be possible to nest this under the `tradeInputRef` so the height calcs include the height of the banner to avoid the odd-looking gap at the bottom:


<img src="https://github.com/user-attachments/assets/885810fb-3ab7-4c96-afc0-2b3b78743ec9" width="200">


Alternatively we should move the quote list to align with the top of the form excluding the banner to keep the layout looking tidy

Reply:
Aligned with the top of the form as a quick win 
<img width="1096" alt="image" src="https://github.com/user-attachments/assets/ca60dcd8-6b9b-4bf3-9c50-58018bfeecb8">

in [30b7cb9](https://github.com/shapeshift/web/pull/8044/commits/30b7cb926fe202f7e1e9e44fd5a3616439f2f4f9)

Reply:
*** SO *** much better